website/pages/en/community.js

```diff
@@ -177,6 +176,26 @@ function Community(props) {
     </CommunityResource>
   );
 
+  const Events = () => (
+    <CommunityResource title={<translate>Events & Meetups</translate>}>
+      <p>
+        <translate>
+          Find recent events and meetups that you love and get together to talk with other developers and researchers.
+        </translate>
+      </p>
+      <p>
+        <Button
+          variant="secondary"
+          size="sm"
```

Comment:
I think you can leave this default size

Reply:
Leave it to be default will make the button size different from other sections' button size.